packages/components/psammead-navigation/src/Dropdown/index.jsx

```diff
@@ -0,0 +1,226 @@
+    display: none;
+  }
+`;
+
+const CrossButton = styled.button`
+  background-color: ${C_EBON};
+  width: 2.75rem;
+  height: 2.75rem;
+  padding: 0;
+  margin: 0;
+  border: 0;
+
+  &:hover,
+  &:focus {
+    box-shadow: inset 0 0 0 0.25rem ${C_WHITE};
+    cursor: pointer;
+  }
+`;
+
+const DropdownUl = styled.ul`
+  list-style-type: none;
+  margin: 0;
+  padding-right: ${GEL_SPACING};
+  padding-left: ${GEL_SPACING};
+  border-bottom: 0.125rem solid ${C_SHADOW};
+`;
+
+const DropdownLi = styled.li`
+  padding-top: 0.75rem;
+  padding-bottom: 0.75rem;
+  border-bottom: 0.0625rem solid ${C_SHADOW};
+
+  &:last-child {
+    padding-bottom: ${GEL_SPACING_DBL};
+    border: 0;
+  }
+`;
+
+const StyledDropdownLink = styled.a`
+  ${({ script }) => script && getPica(script)};
+  ${({ service }) => service && getSansRegular(service)}
+  color: ${C_WHITE};
+  text-decoration: none;
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+    cursor: pointer;
+  }
+`;
+
+const StyledCurrentLink = styled.span`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `border-left: ${GEL_SPACING_HLF} solid ${C_WHITE};
+        padding-left: ${GEL_SPACING};`
+      : `border-right: ${GEL_SPACING_HLF} solid ${C_WHITE};
+        padding-right: ${GEL_SPACING};`}
+`;
+
+export const DropdownNavigationLi = ({
+  children,
+  script,
+  currentPageText,
+  active,
+  service,
+  url,
+  dir,
+}) => (
+  <DropdownLi role="listitem">
+    {active && currentPageText ? (
+      <StyledDropdownLink script={script} service={service} href={url}>
+        <StyledCurrentLink
+          dir={dir}
+          // eslint-disable-next-line jsx-a11y/aria-role
+          role="text"
+        >
+          <VisuallyHiddenText>{currentPageText}</VisuallyHiddenText>
+          {children}
+        </StyledCurrentLink>
+      </StyledDropdownLink>
+    ) : (
+      <StyledDropdownLink script={script} service={service} href={url}>
+        {children}
+      </StyledDropdownLink>
+    )}
+  </DropdownLi>
+);
+
+export const AmpDropdown = ({ announcedText, closeAction, children }) => (
+  <DropdownWrapper>
+    <CrossButton
+      aria-label={announcedText}
+      on={closeAction}
+      // eslint-disable-next-line react/jsx-boolean-value
+      aria-expanded={true}
+    >
+      {navigationIcons.cross}
+    </CrossButton>
+    <DropdownUl role="list">{children}</DropdownUl>
+  </DropdownWrapper>
+);
+
+export const CanonicalDropdown = ({ announcedText, closeAction, children }) => (
+  <DropdownWrapper>
+    <CrossButton
+      aria-label={announcedText}
+      onClick={closeAction}
+      // eslint-disable-next-line react/jsx-boolean-value
+      aria-expanded={true}
+    >
+      {navigationIcons.cross}
+    </CrossButton>
+    <DropdownUl role="list">{children}</DropdownUl>
+  </DropdownWrapper>
+);
+
+const HamburgerButton = styled.button`
+  width: 2.75rem;
+  height: 2.75rem;
+  background-color: ${C_POSTBOX};
+  padding: 0;
+  margin: 0;
+  border: 0;
+
+  &:hover,
```

Comment:
Do we need to have a `hover` styling considering this will be only visible on mobile devices?

Reply:
Zeplin specified styling for `hover`&`focus` so I went along with it, but I guess since it's for mobile devices we don't need it. Will delete those now.